plugins/tiddlywiki/webdav/plugin.info

```diff
@@ -0,0 +1,7 @@
+{
+	"title": "$:/plugins/tiddlywiki/webdav",
+	"description": "WebDAV sync adaptor",
+	"author": "Sebastian Silva",
+	"core-version": ">=5.0.0",
```

Comment:
There are no supported 5.0.0 versions ... TW [started with 5.1.0](https://tiddlywiki.com/#Releases) excluding alphas and betas.

Did you test the plugin with 5.1.0? ... It would be cool if it would work with older versions. BUT we have to be sure. Otherwise we will get a lot of support requests!

Reply:
This is what almost all core plugins specify. But still a good thing to change :)